ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```
@@ -531,3 +531,17 @@ ethernet_interfaces:
         reauth_timeout_ignore: true
         tx_period: 10
       reauthorization_request_limit: 2
+
+  Ethernet46:
+    description: IP Helper
+    type: routed
+    ip_address: 172.31.255.1/31
+    ip_helpers:
+      - ip_helper: 10.10.96.101
+        source_interface: Loopback0
+      - ip_helper: 10.10.96.151
+        vrf: MGMT
+      - ip_helper: 10.10.96.150
+        source_interface: Loopback0
+        vrf: MGMT
+      - ip_helper: 10.10.64.150
```

```suggestion
    ip_helpers:
      10.10.96.101:
        source_interface: Loopback0
      10.10.96.151:
        vrf: MGMT
      10.10.96.150:
        source_interface: Loopback0
        vrf: MGMT
      10.10.64.150:
```

Sorry to get you back to the same example again. Please create the example below in **eos_cli_config_gen_v4.0/host_vars/ethernet-interface.yml** and run molecule for that scenario:

```
ip_helpers:
      - ip_helper: 10.10.96.101
        source_interface: Loopback0
      - ip_helper: 10.10.96.151
        vrf: MGMT
      - ip_helper: 10.10.96.150
        source_interface: Loopback0
        vrf: MGMT
      - ip_helper: 10.10.64.150
```